packages/services/f-http/src/createClient.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import defaultOptions from './defaultOptions';
+
+let _configuration = null;
+let _axiosInstance = null;
+
+/**
+    * Get a resource
+    * @param {string} resource - The resource to post (URL)
+    * @return {object} - Returns data from response
+*/
+const get = async resource => {
+    try {
+        const response = await _axiosInstance.get(resource);
+
+        return response.data;
+    } catch (error) {
+        if (_configuration.errorCallback) {
+            _configuration.errorCallback(error, resource);
+        }
+
+        throw error;
+    }
+};
+
+/**
+    * Post a resource
+    * @param {string} resource - The resource to post (URL)
+    * @param {object} body - The request body, contents of the resource
+    * @param {object} headers - Any additional request headers you want to post
+    * @return {object} - Returns data from response
+*/
+const post = async (resource, body, headers = {}) => {
+    try {
+        const config = {
+            headers
+        };
+
+        const response = await _axiosInstance.post(resource, body, config);
+
+        return response.data;
+    } catch (error) {
+        if (_configuration.errorCallback) {
+            _configuration.errorCallback(error, resource);
+        }
+
+        throw error;
```

Comment:
It seems that this snippet will be repeated pretty much across all the different request types, so going forward you might benefit from extracting this into a separate function.

Reply:
Thats the plan

Reply:
Extracted and tested